crates/forge/tests/e2e/running.rs

```diff
@@ -727,9 +728,11 @@ fn validate_init(temp: &TempDir, validate_snforge_std: bool) {
 
             [scripts]
             test = "snforge test"
+            {SCARB_MANIFEST_TEMPLATE_CONTENT}
```

Comment:
Isn't this missing adding these contents?

Reply:
Resolved offline.